app/assets/javascripts/application.js

```diff
@@ -395,11 +402,27 @@ function spinner_placeholder(text){
 }
 
 function notify(item, type) {
-  var options = { type: type, sticky: (type != 'success') };
-  $.jnotify($(item).text(), options);
+  var icon = typeToIcon(type);
+  var options = { classMessage: "alert alert-"+type,
+                  classBackground: "",
+                  type: type,
+                  sticky: (type != 'success') };
+  $.jnotify("</div>" + icon + $(item).text(), options);
```

Comment:
There is no way to close the notification popup. This means that the right most menus (user&administer) can't be accessed without browsing to a different page first.